src/pip/_internal/utils/misc.py

```diff
@@ -877,3 +877,18 @@ def enum(*sequential, **named):
     reverse = dict((value, key) for key, value in enums.items())
     enums['reverse_mapping'] = reverse
     return type('Enum', (), enums)
+
+
+def confirm_dependencies(req):
+    installed_distributions = get_installed_distributions()
+    dep_keys = set()
+    for dist in installed_distributions:
+        dep_keys.update(
+            [dist for d in dist.requires() if d.key == req.req.name])
+    if not dep_keys:
+        return True
+    logger.info("%s is depended from:" % req.req)
```

Comment:
The wording of this message "REQ is depended from: ..." confused me. I assume what it means is that the following packages depend on REQ, so would it be better to word the message something like the following?

```python
    logger.info("The following packages depend on %s and may break if it is uninstalled:" % req.req)
```